package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "es6-promise": "3.2.1",
     "mongodb-core": "2.1.10",
+    "mongodb-core": "christkv/mongodb-core#2.0",
```

Comment:
On purpose it's so travis builds against the latest mongodb-core branch. Don't worry it's not in prod :)